src/main/java/com/google/sps/data/MapLocation.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+/** Class representing a MapLocation as a coordinate **/
```

Comment:
nit: You can reduce this to "MapLocation representing coordinate values"

Reply:
Got it